tests/test_D1_model_components.py

```diff
@@ -333,17 +333,19 @@ def helper_test_sink_in_model_and_dict(
         # foreach input bus - these values are expected to be different
         # depending on `dispatchable`
         if amount_inputs == 1:
-            input_bus_names = [dict_asset[INPUT_BUS_NAME]]
+            inflow_direction_s = [dict_asset[INFLOW_DIRECTION]]
             if dispatchable is True:
                 dispatch_price = [dict_asset[DISPATCH_PRICE][VALUE]]
         elif amount_inputs > 1:
-            input_bus_names = dict_asset[INPUT_BUS_NAME]
+            inflow_direction_s = dict_asset[INFLOW_DIRECTION]
             if dispatchable is True:
                 dispatch_price = dict_asset[DISPATCH_PRICE][VALUE]
         else:
             raise ValueError("`amount_inputs` should be int but not zero.")
-        for input_bus_name, i in zip(input_bus_names, range(len(input_bus_names))):
-            input_bus = self.model.entities[-1].inputs[self.busses[input_bus_name]]
+        for inflow_direction, i in zip(
+            inflow_direction_s, range(len(inflow_direction_s))
+        ):
```

Comment:
```suggestion
        for i, inflow_direction in enumerate(inflow_direction_s):
```


for this you can use the builtin function `enumerate`

Reply:
e90ae43e881ef267a428720ac79b6a151e4726c0